awx/main/migrations/0009_v330_upgrade.py

```diff
@@ -0,0 +1,147 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.7 on 2017-11-08 20:35
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('main', '0008_v320_drop_v1_credential_fields'),
+    ]
+
+    operations = [
+        migrations.AlterField(
```

Comment:
What actually changed with these `created_by`/`modified_by` fields as part of the upgrade?

Reply:
I had this question too but it seems to have vanished above...